toDoApp/src/navigation/routes.ts

```diff
@@ -0,0 +1,12 @@
+import { routesType } from '../navigation/types'
+
+export const constants = {
+  taskTitle: 'Task title',
+  taskDescription: 'Task description',
+}
```

Comment:
This is something pretty related to the screen itself right? You need to maintain things that belongs together in the same folder.